File: Python/Ejercicio 5/main.py
acumulador = 0
contador = 0
for i in range(10):
    nua = int(input("ingrese una temperatura"))
    if(nua > 5 - nua < 15):
        acumulador = acumulador + nua
        contador = contador + 1

promedio = (acumulador/contador)
print("el promedio de la temperatura es: ", promedio)

File: C++/Ejercicio 5/main.cpp
#include <iostream>

using namespace std;

int main()
{

int indice, contador =0;

double promedio, temperatura, acumulador;

for(indice = 0 ; indice < 10; indice ++)
{
cout << " ingrese un numero " << endl;
cin >> temperatura;

if (temperatura > 5 && temperatura <15)
{
acumulador += temperatura;
contador++;
}


}




promedio = double (acumulador)/contador;
          //cast estatico
cout << "promedio de temperatura " << promedio;

return 0;
}
